### PR TITLE
feat: add builder files

### LINKED ISSUE #36: Add builder translations

### PR DESCRIPTION
**Pull request by pyphilia**
close #36

### CODE REVIEW

File: src/langs/en/uppy.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "closeModal": "Close Modal",
+  "addMoreFiles": "Add more files",
+  "addingMoreFiles": "Adding more files",
+  "importFrom": "Import from %{name}",
```
Comment (spaenleh) [inline src/langs/en/uppy.json:5]: Is this file used by uppy directly ? So this is why the template string is different ?

File: src/langs/fr/builder.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
+{
+  "ACCESS_ITEM_LOGIN_TAG_TOOLTIP": "Cet élément permet de s'y connecter directement. Les nouveaux utilisateurs peuvent accéder à cet élément s'ils utilisent cette authentification.",
+  "ACCESS_ITEM_LOGIN_TAG": "N'importe quel visiteur authentifié avec le lien",
+  "ACCESS_PUBLIC_TAG_TOOLTIP": "Cet élément est public. N'importe qui peut y accéder.",
+  "ACCESS_PUBLIC_TAG": "Public",
+  "ACCESS_TITLE": "Accès",
+  "ANALYTICS_DASHBOARD_LABEL": "Dashboard des Analytiques",
+  "COLLAPSE_ITEM_COLLAPSE_TEXT": "Collapse",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:8]: Peut être utiliser:
```suggestion
  "COLLAPSE_ITEM_COLLAPSE_TEXT": "Minifier",
```

File: src/langs/fr/builder.json [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  "COLLAPSE_ITEM_UNCOLLAPSE_TEXT": "Uncollapse",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:9]:
```suggestion
  "COLLAPSE_ITEM_UNCOLLAPSE_TEXT": "Maximiser",
```

File: src/langs/fr/builder.json [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  "COPY_ITEM_MODAL_TITLE": "Où copier cet élément?",
+  "CREATE_ITEM_ADD_BUTTON": "Ajouter",
+  "CREATE_ITEM_APP_TITLE": "Créer une App",
+  "CREATE_ITEM_DOCUMENT_TITLE": "Créer un Document",
+  "CREATE_ITEM_FOLDER_TITLE": "Créer un Dossier",
+  "CREATE_ITEM_LINK_LABEL": "Lien",
+  "CREATE_ITEM_LINK_TITLE": "Créer un Lien",
+  "CREATE_NEW_ITEM_NAME_LABEL": "Nom",
+  "CREATE_SHORTCUT_DEFAULT_NAME": "Shortcut vers {{name}}",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:18]:
```suggestion
  "CREATE_SHORTCUT_DEFAULT_NAME": "Raccourci vers {{name}}",
```

File: src/langs/fr/builder.json [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  "CROP_IMAGE_MODAL_CONTENT_TEXT": "Couper l'image choisie pour correspondre aux critères d'image.",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:19]:
```suggestion
  "CROP_IMAGE_MODAL_CONTENT_TEXT": "Couper l'image choisie pour correspondre aux critères de taille d'image.",
```

File: src/langs/fr/builder.json [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  "FAVORITE_ITEM_REMOVE_TEXT": "Retirer des favoris",
+  "FAVORITE_ITEMS_TITLE": "Eléments favoris",
+  "FLAG_ITEM_MODAL_TITLE": "Signaler l'élément",
+  "FLAG_ITEM_REASON_TITLE": "Sélectionner la raison du signalement de cet élément",
+  "HIDE_ITEM_HIDE_TEXT": "Cacher",
+  "HIDE_ITEM_SHOW_TEXT": "Montrer",
+  "HOME_TITLE": "Accueil",
+  "IMPORT_H5P_INFORMATIONS": "Une application H5P peut être téléchargée avec un fichier .h5p (ex: depuis H5P.com, des services externes comme Moodle, etc).",
+  "IMPORT_H5P_WARNING": "Une fois que le fichier est accepté, cela prendra quelques minutes avant qu'il ne soit disponible.",
+  "IMPORT_ZIP_INFORMATION": "Pour télécharger vos fichiers depuis la plateforme graasp.eu, faites un clique droit et choisissez \"Télécharger en tant que ZIP\".",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:44]:
```suggestion
  "IMPORT_ZIP_INFORMATION": "Pour télécharger vos fichiers depuis la plateforme graasp.eu, faîtes un clique droit et choisissez \"Télécharger en tant que ZIP\".",
```

File: src/langs/fr/builder.json [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+  "LIBRARY_SETTINGS_PUBLISH_NOTIFICATIONS_LABEL": "Envoyer une notification par email à tous les co-éditeurs",
+  "LIBRARY_SETTINGS_PUBLISHED_STATUS": "Cet élément est publié. N'importe qui peut y accéder et l'élément est répertorié sur Graasp Library, notre librarie de ressources d'apprentissage.",
+  "LIBRARY_SETTINGS_VALIDATION_CONFIGURATION_INFORMATIONS": "Choisissez les options pour votre ressource pour la rendre facilement accessible pour les autres.",
+  "LIBRARY_SETTINGS_VALIDATION_INFORMATIONS": "Vous avez besoin de valider votre élément avant de le publier. Attendez quelques minutes pour que la validation se termine.",
+  "LIBRARY_SETTINGS_VALIDATION_PUBLICATION_INFORMATIONS": "Une fois que votre élément est validé, vous pouvez publier votre élément en cliquant sur le bouton.",
+  "LIBRARY_SETTINGS_VALIDATION_PUBLICATION_TITLE": "Publications",
+  "LIBRARY_SETTINGS_VALIDATION_STATUS_FAILURE": "Votre élément pourrait contenir du contenu inapproprié. Supprimez-les et validez à nouveau votre élément. Si vous avez des question, contactez {{contact, string}}.",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:100]:
```suggestion
  "LIBRARY_SETTINGS_VALIDATION_STATUS_FAILURE": "Votre élément pourrait présenter des contenus inapproprié. Supprimez-les et validez à nouveau votre élément. Si vous avez des question, contactez {{contact, string}}.",
```

File: src/langs/fr/builder.json [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+  "LIBRARY_SETTINGS_VALIDATION_STATUS_PENDING_AUTOMATIC": "Votre élément est en train d'être validé. Une fois réussie, vous pourrez publier votre élément.",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:101]:
```suggestion
  "LIBRARY_SETTINGS_VALIDATION_STATUS_PENDING_AUTOMATIC": "Votre élément est en cours de validation. Une fois réussie, vous pourrez publier votre élément.",
```

File: src/langs/fr/builder.json [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+  "PIN_ITEM_UNPIN_TEXT": "Détacher",
+  "PLAY_BUTTON_TOOLTIP": "Montrer la Vue Player",
+  "RECYCLE_ITEM_BUTTON": "Mettre à la poubelle",
+  "SETTINGS_LINK_SETTINGS_INFORMATIONS": "Si les deux paramètres sont désactivés, un bouton sera affiché par défaut.",
+  "SETTINGS_LINK_SETTINGS_TITLE": "Paramètres du lien",
+  "SETTINGS_LINK_SHOW_BUTTON": "Montrer le bouton cliquable",
+  "SETTINGS_LINK_SHOW_IFRAME": "Montrer le lien dans une iframe",
+  "SETTINGS_PIN_ITEM_LABEL": "Epingler",
+  "SETTINGS_SHOW_CHAT_LABEL": "Afficher le chat",
+  "SETTINGS_THUMBNAIL_SETTINGS_INFORMATIONS": "Envoyer une image pour mettre à jour votre icone. Recharger la page pour visualiser les changements.",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:123]:
```suggestion
  "SETTINGS_THUMBNAIL_SETTINGS_INFORMATIONS": "Importez une image pour mettre à jour votre photo de profil. Rechargez la page pour visualiser les changements.",
```

File: src/langs/fr/builder.json [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+  "SETTINGS_THUMBNAIL_TITLE": "Icone",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:124]:
```suggestion
  "SETTINGS_THUMBNAIL_TITLE": "Photo de profil",
```

File: src/langs/fr/builder.json [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+  "SHARING_AUTHORIZED_MEMBERS_EMPTY_MESSAGE": "Aucun utilisateur ne peut accéder à cet élément.",
+  "SHARING_AUTHORIZED_MEMBERS_TITLE": "Membres Autorisés",
+  "SHARING_INVITATIONS_EMPTY_MESSAGE": "Aucune invitation pour cet élément.",
+  "SHARING_INVITATIONS_RESEND_BUTTON": "Renvoyer l'Invitation",
+  "SHARING_INVITATIONS_TITLE": "Invitations En Cours",
+  "SHARING_TITLE": "Partage",
+  "TABLE_ASC_SORT_LABEL": "trié par ordre croissant",
+  "TABLE_DESC_SORT_LABEL": "trié par ordre décroissant",
+  "TABLE_SELECT_ALL_LABEL": "Sélectioner Tous les Eléments",
+  "THUMBNAIL_SETTING_MY_THUMBNAIL_ALT": "icone actuelle",
```
Comment (spaenleh) [inline src/langs/fr/builder.json:148]:
```suggestion
  "THUMBNAIL_SETTING_MY_THUMBNAIL_ALT": "photo de profil actuelle",
```

**Review by spaenleh — APPROVED** [review]
Looks good to me ! I left some suggestions and questions 🎃

File: src/langs/en/uppy.json (hunk 1 above)
Comment (pyphilia) [reply src/langs/en/uppy.json:5]: Exactly!

**Pull request merged**